StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/MessageSendable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  MessageSendable.swift
```

Comment:
Nit: Filename should probably match the type
```suggestion
//  MessageSender.swift
```